Traceback (most recent call last):
  File "/home/l.peiwang/miniconda3/envs/pasta_env_new_2025/bin/pip", line 11, in <module>
    sys.exit(main())
  File "/home/l.peiwang/.local/lib/python3.10/site-packages/pip/_internal/cli/main.py", line 77, in main
    command = create_command(cmd_name, isolated=("--isolated" in cmd_args))
  File "/home/l.peiwang/.local/lib/python3.10/site-packages/pip/_internal/commands/__init__.py", line 114, in create_command
    module = importlib.import_module(module_path)
  File "/home/l.peiwang/miniconda3/envs/pasta_env_new_2025/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/l.peiwang/.local/lib/python3.10/site-packages/pip/_internal/commands/install.py", line 15, in <module>
    from pip._internal.cli.req_command import (
  File "/home/l.peiwang/.local/lib/python3.10/site-packages/pip/_internal/cli/req_command.py", line 20, in <module>
    from pip._internal.index.collector import LinkCollector
  File "/home/l.peiwang/.local/lib/python3.10/site-packages/pip/_internal/index/collector.py", line 37, in <module>
    from pip._internal.network.session import PipSession
  File "/home/l.peiwang/.local/lib/python3.10/site-packages/pip/_internal/network/session.py", line 32, in <module>
    from pip._vendor.cachecontrol import CacheControlAdapter as _BaseCacheControlAdapter
ModuleNotFoundError: No module named 'pip._vendor.cachecontrol'
Traceback (most recent call last):
  File "/home/l.peiwang/miniconda3/envs/pasta_env_new_2025/bin/pip", line 11, in <module>
    sys.exit(main())
  File "/home/l.peiwang/.local/lib/python3.10/site-packages/pip/_internal/cli/main.py", line 77, in main
    command = create_command(cmd_name, isolated=("--isolated" in cmd_args))
  File "/home/l.peiwang/.local/lib/python3.10/site-packages/pip/_internal/commands/__init__.py", line 114, in create_command
    module = importlib.import_module(module_path)
  File "/home/l.peiwang/miniconda3/envs/pasta_env_new_2025/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/l.peiwang/.local/lib/python3.10/site-packages/pip/_internal/commands/install.py", line 15, in <module>
    from pip._internal.cli.req_command import (
  File "/home/l.peiwang/.local/lib/python3.10/site-packages/pip/_internal/cli/req_command.py", line 20, in <module>
    from pip._internal.index.collector import LinkCollector
  File "/home/l.peiwang/.local/lib/python3.10/site-packages/pip/_internal/index/collector.py", line 37, in <module>
    from pip._internal.network.session import PipSession
  File "/home/l.peiwang/.local/lib/python3.10/site-packages/pip/_internal/network/session.py", line 32, in <module>
    from pip._vendor.cachecontrol import CacheControlAdapter as _BaseCacheControlAdapter
ModuleNotFoundError: No module named 'pip._vendor.cachecontrol'
Traceback (most recent call last):
  File "/home/l.peiwang/aagn/train.py", line 10, in <module>
    from dataset import ADNIDataset
  File "/home/l.peiwang/aagn/dataset.py", line 1, in <module>
    import ants
ModuleNotFoundError: No module named 'ants'
